tests/unit/dataactvalidator/test_scripts_loadTas.py

```diff
@@ -24,11 +29,12 @@ def test_loadTas_multiple(database, tmpdir):
     """If we have two rows in the CSV, we should have two TASLookups"""
     results = import_tas(
         database.session, tmpdir,
-        {'ATA': 'aaa', 'AID': 'bbb', 'A': 'ccc', 'BPOA': 'ddd', 'EPOA': 'eee',
-         'MAIN': 'ffff', 'SUB': 'ggg'},
-        {'ATA': '111', 'AID': '222', 'A': '333', 'BPOA': '444', 'EPOA': '555',
-         'MAIN': '6666', 'SUB': '777'}
+        {'ACCT_NUM': '6', 'ATA': 'aaa', 'AID': 'bbb', 'A': 'ccc',
```

Comment:
No doubt I'm missing something obvious, but I'm confused about what we're testing here. Is this test to ensure that the helper `import_tas` function works as expected before it's used in other tests? Does `import_tas` need to read the .csv you're creating using pandas to make sure all the NULLs and spaces arrive in the expected way?